cellfinder/core/classify/resnet.py

```diff
@@ -133,6 +134,10 @@ def non_residual_block(
     )(x)
     x = BatchNormalization(axis=axis, epsilon=bn_epsilon, name="conv1_bn")(x)
     x = Activation(activation, name="conv1_activation")(x)
+
+    if keras.config.backend() == "torch":
+        pooling_padding = "valid"
```

Comment:
I'm not the biggest fan of this because then it always overwrites the `pooling_padding` function parameter, whereas we only care to change it if the backend is torch and the padding is set to `same`. However, if we're just supporting torch, perhaps the default function parameter can just be changed to `valid`?